source/manual/howto-switch-off-app.html.md

```diff
@@ -4,7 +4,7 @@ title: Switch an app off temporarily
 layout: manual_layout
```

Comment:
Is this one actually useful?

Reply:
I think it's easy to forget to disable puppet, but whether or not people would actually look at this page, I'm not sure.

Reply:
Yeah, that's why I'm wondering if we really need it.